raiden-dapp/tests/unit/components/address-display.spec.ts

```diff
@@ -1,4 +1,4 @@
-import { $identicon } from '../utils/mocks';
+import { $t } from '../utils/mocks';
```

Comment:
I've seen this a multiple times. Is it so that the identicon is **not** needed and imported anymore or did you just rename the `$identicon` to `$t`?

Reply:
The identicon is only necessary for a very few components where it is actually used. While the translation function is basically used everywhere.